### DISCUSSION TITLE
Galaxy Badge

### DISCUSSION THREAD

**Discussion by Saqib198** [Schweinepriester/github-profile-achievements#916] [General]
To unlock the Galaxy Brain badge on GitHub, you need to answer at least 2 discussions. The discussions must be in a public repository, and you cannot mark your own answers to your own questions.

**Comment by aaraifkhan** [comment]
Hain, Never heard of this, will try to use GitHub more to achieve badges from now on...

**Comment by 17351269** [comment]
Mh

در تاریخ شنبه 25 مه 2024، 10:45 aaraifkhan ***@***.***> نوشت:
